src/overlay005/field_control.c

```diff
@@ -0,0 +1,1092 @@
+        input->endMovement = TRUE;
+    }
+
+    if (playerDir == DIR_NORTH && heldKeys & PAD_KEY_UP || 
+        playerDir == DIR_SOUTH && heldKeys & PAD_KEY_DOWN || 
+        playerDir == DIR_WEST  && heldKeys & PAD_KEY_LEFT || 
+        playerDir == DIR_EAST  && heldKeys & PAD_KEY_RIGHT) {
+
+        input->transitionDir = playerDir;
+    } else {
+        input->transitionDir = DIR_NONE;
+    }
+
+    input->playerDir = sub_02061308(fieldSystem->playerAvatar, pressedKeys, heldKeys);
+}
+
+int FieldInput_Process (const FieldInput *input, FieldSystem *fieldSystem)
+{
+    if (input->unused5 == FALSE && sub_0203F5C0(fieldSystem, 1) == TRUE) {
+        return TRUE;
+    }
+
+    if (input->unused5 == FALSE) {
+        BOOL doublesCheck = sub_02054AB0(Party_GetFromSavedata(fieldSystem->saveData));
+
+        if (sub_0206A984(SaveData_Events(fieldSystem->saveData)) == TRUE) {
+            doublesCheck = TRUE;
+        }
+
+        if (sub_02067A84(fieldSystem, doublesCheck) == TRUE || 
+            sub_02071CB4(fieldSystem, 2) == TRUE && 
+            ov8_0224C51C(fieldSystem) == TRUE) {
+
+            sub_0205F56C(fieldSystem->playerAvatar);
+            MapObjectMan_PauseAllMovement(fieldSystem->mapObjMan);
+            return TRUE;
+        }
+    }
+
+    if (input->movement) {
+        sub_0206A9A4(SaveData_Events(fieldSystem->saveData));
+
+        if (Field_ProcessStep(fieldSystem) == TRUE) {
+            return TRUE;
+        }
+    }
+
+    if (input->unused5 == FALSE) {
+        int playerEvent = PLAYER_EVENT_NONE;
+        int direction = sub_02061308(fieldSystem->playerAvatar, input->pressedKeys, input->heldKeys);
+
+        if (inline_0204E650_2(SaveData_Events(fieldSystem->saveData))) {
+            playerEvent |= PLAYER_EVENT_USED_STRENGTH;
+        }
+
+        if (sub_020549A0(Party_GetFromSavedata(fieldSystem->saveData), 127) != 0xff) {
+            playerEvent |= PLAYER_EVENT_USED_WATERFALL;
+        }
+
+        if (sub_02071CB4(fieldSystem, 9) == TRUE && ov9_02250F74(fieldSystem) == TRUE) {
+            playerEvent |= PLAYER_EVENT_DISTORTION_WORLD;
+        }
+
+        if (ov5_021DFDE0(fieldSystem, fieldSystem->playerAvatar, direction, playerEvent) == TRUE) {
+            return TRUE;
+        }
+    }
+
+    if (input->endMovement) {
+        if (Field_CheckWildEncounter(fieldSystem)) {
+            return TRUE;
+        }
+
+        if (Field_CheckSign(fieldSystem) == TRUE) {
+            return TRUE;
+        }
+
+        if (sub_02071CB4(fieldSystem, 9) == TRUE) {
+            int direction = (s8)input->playerDir;
+
+            if (direction == DIR_NONE) {
+                direction = PlayerAvatar_GetDir(fieldSystem->playerAvatar);
+            }
+
+            if (ov9_0224A67C(fieldSystem, direction) == TRUE) {
+                return TRUE;
+            }
+        }
+    }
+
+    if (input->sign && PlayerAvatar_GetDir(fieldSystem->playerAvatar) == input->playerDir) {
+        if (Field_CheckSign(fieldSystem) == TRUE) {
```

Comment:
```suggestion
    if (input->sign
        && PlayerAvatar_GetDir(fieldSystem->playerAvatar) == input->playerDir
        && FieldEventCheck_Sign(fieldSystem) == TRUE) {
```

Reply:
I think I've seen elsewhere in the repo that the operators go on the end of the line instead of the beginning in multiline conditionals. Do we have a standard for that?

Reply:
Ideally, continuations should start with an operator of some sort to clearly delineate that they belong to a continuation and aren't just indented to be indented.